sdk/spring/README.md

```diff
@@ -0,0 +1,83 @@
+# Azure Spring Boot client library for Java
+
+## Getting started
+### Introduction
+
+This repo is for Spring Boot Starters of Azure services. It helps Spring Boot developers to adopt Azure services.
+
+### Support Spring Boot
+This repository supports both Spring Boot 1.5.x, 2.0.x and 2.1.x. Please read [Branch and release](https://github.com/Microsoft/azure-spring-boot/wiki/Branch-and-release) for branch mapping.
```

Comment:
Can we move this wiki content to the azure-sdk-for-java wiki? Ping me and I will give guidance about where to put this content.

Reply:
Moved.